target_snowflake/__init__.py

```diff
@@ -257,6 +277,27 @@ def persist_lines(config, lines, table_cache=None, file_format_type: FileFormatT
                 else:
                     stream_to_sync[stream] = DbSync(config, o, table_cache, file_format_type)
 
+                if archive_load_files_enabled:
```

Comment:
```suggestion
                if archive_load_files:
```

can we change it to be single level option?